### DISCUSSION TITLE
Does the Fabric GUI work on Windows?

### DISCUSSION THREAD

**Discussion by CPAtoCybersecurity** [danielmiessler/Fabric#421] [Q&A]
Passing along a question here that I didn't know the answer to https://youtu.be/aEdS8ocmOZs.  Thanks.

**Comment by Gerkinfeltser** [comment]
It does! I did have to do a bit of manual tweaking to a function to make it work though. Your mileage may vary but see my workaround here:
https://github.com/danielmiessler/fabric/issues/369#issuecomment-2119288522
